goguen/formal-spec/utxo.tex

```diff
@@ -542,117 +450,17 @@ \subsection{Two-Phase Transaction Validation for Non-Native Scripts}
 result in different validation outcomes running the same script on the same
 arguments.
 
-
 \begin{figure}[htb]
-  \begin{equation}
-    \inference[Scripts-Val]
-    {
-    \fun{txvaltag}~\var{tx} = \True  &
-    \var{remExU}~\geq~0
-    }
-    {
-    \begin{array}{l}
-      \var{pp}\\
-      \var{tx}\\
-    \end{array}
-      \vdash
-      \left(
-      \begin{array}{r}
-        \var{remExU}\\
-      \end{array}
-      \right)
-      \trans{sval}{\epsilon}
-      \left(
-      \begin{array}{r}
-        \var{remExU}\\
-      \end{array}
-      \right) \\
-    }
-  \end{equation}
-  \begin{equation}
-    \inference[Scripts-Stop]
-    { \\~\\
-    (\var{isVal},\var{remExU'})~:=~ \llbracket sc \rrbracket_
-    {cm,\var{remExU}} dt \\
-    (sc, dt, cm) := s
-    \\
-    ~
-    \\
-    \fun{txvaltag}~\var{tx} = \False &
-    (\var{remExU'}~<~0 ~ \lor ~ \var{isVal} = \False)
-    }
-    {
-    \begin{array}{l}
-      \var{pp}\\
-      \var{tx}\\
-    \end{array}
-      \vdash
-      \left(
-      \begin{array}{r}
-        \var{remExU}\\
-      \end{array}
-      \right)
-      \trans{sval}{\Gamma;s}
-      \left(
-      \begin{array}{r}
-        \var{remExU}\\
-      \end{array}
-      \right)
-    }
-  \end{equation}
-  \begin{equation}
-    \inference[Scripts-Ind]
-    {
-    {
-    \begin{array}{l}
-      \var{pp}\\
-      \var{tx}\\
-    \end{array}
-    }
-      \vdash
-      \left(
-      {
-      \begin{array}{r}
-        \var{remExU}\\
-      \end{array}
-      }
-      \right)
-      \trans{sval}{\Gamma}
-      \left(
-      {
-      \begin{array}{r}
-        \var{remExU'}\\
-      \end{array}
-      }
-      \right) \\
-    (\var{isVal},\var{remExU''})~:=~ \llbracket sc \rrbracket
-    _{cm,\var{remExU'}} dt \\
-    (sc, dt, cm) := s & \var{remExU''}~\geq~0
-    }
-    {
-    \begin{array}{l}
-      \var{pp}\\
-      \var{tx}\\
-    \end{array}
-      \vdash
-      \left(
-      \begin{array}{r}
-        \var{remExU}\\
-      \end{array}
-      \right)
-      \trans{sval}{\Gamma;s}
-      \left(
-      \begin{array}{r}
-        \varUpdate{remExU''}\\
-      \end{array}
-      \right)
-    }
-  \end{equation}
-  \caption{Script validation rules}
-  \label{fig:rules:utxo-scrval}
+  \begin{align*}
+    & \fun{evalScripts} \in \seqof{(\ScriptPurpose \times \Script \times \seqof{Data})} \to \ExUnits \to \Bool \\
```

Comment:
Is the type of the first parameter supposed to be `ScriptPlutus x Data* x CostMod`? (I thought I left this comment earlier, not sure what happened...)

Reply:
Yeah, I don't know what happened here, but I just found this while fixing evalScripts as well. Just pushed the fix